packages/hardhat-core/src/internal/hardhat-network/provider/fork/ForkTransaction.ts

```diff
@@ -1,11 +1,15 @@
+import { Transaction, TxData, TxOptions } from "@ethereumjs/tx";
```

Comment:
Left here

Reply:
Note: have left this extending "LegacyTransaction" for the moment - not certain what to do about 2930 txs for this (or for FakeTransaction) atm.

Reply:
That was actually a note to myself: I reviewed up to this file 😅

I don't have a clear picture of what to do with 2930 yet.